torch/csrc/jit/tensorexpr/bounds_inference.cpp

```diff
@@ -96,6 +106,14 @@ BoundsInfo getInferredBounds(
       analyzer.accessesWithin(s), getAllBufs(s), distinctAccessKinds);
 }
 
+BoundsInfo getInferredBounds(
```

Comment:
This is a variant of the above function, but for an `Expr`. This is needed to get the access bounds in a `Load`.